config.ts

```diff
@@ -39,18 +43,22 @@ export const config = {
     showDebug: true,
     l1: {
       chain: goerli,
+      chainLogo: '/eth.svg',
       wormholeChainId: 2,
       tokenAddress: '0x449FdC2D5038E0447801025e8Eb47DD8b68a7B90' as `0x${string}`,
       tokenDecimals: 18,
+      tokenLogo: '/poolTogether.svg',
       erc20Bridge: '0x46A5D1C7c2eF4d276562DB6f504b25bD88CfeeD2' as `0x${string}`,
       governor: '0xA4c74075770582ad5873E3d9e288f7877BD90c85' as `0x${string}`,
       metadataBridge: '0x49F8948E4455Eb1d8D3964a9DC17f831C12B0893' as `0x${string}`,
     },
     l2: {
       chain: optimismGoerli,
+      chainLogo: '/optimism.svg',
       wormholeChainId: 24,
       tokenAddress: '0x08841d6c288EE8b2f25b58469492CF65701d0414' as `0x${string}`,
       tokenDecimals: 18,
+      tokenLogo: '/poolTogether.svg',
```

Comment:
[rr] this looks good; so how about `logoUri`? it seems like these 2 properties represent the same thing. should we refactor away from `logoUri`?